modules/base/rendering/renderablesphere.cpp

```diff
@@ -145,10 +159,11 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
-    // This `Renderable` represents a simple sphere with an image. The image that is shown
-    // should be in an equirectangular projection/spherical panoramic image or else
-    // distortions will be introduced. The `Orientation` parameter determines whether the
-    // provided image is shown on the inside, outside, or both sides of the sphere.
+    // This `Renderable` represents a simple sphere with an image. Per default, the
+    // sphere uses an equirectangular projection for the image mapping.
+    //
+    // The `Orientation` parameter determines whether the provided image is shown on
```

Comment:
Should this be here? I actually kind of like it but it will be duplicated since it is a property in itself and will have its documentation further down the page already.  Although it looks like we used to have it here before as well 🤷‍♀️

Reply:
Yeah we used to have it here before, so I just kept it. Also in general I think it's nice to highlight some important properties/arguments/members in the main documentation string, so having it here does not hurt :) We do this for e.g. [RenderablePotinCloud](https://docs.openspaceproject.com/latest/reference/asset-components/Renderable/RenderablePointCloud.html) as well

However, the docs for `RenderableSphere` is not actually visible anywhere on docs.openspaceproject.com, so it doesn't matter much what is in this documentation at the moment 😅 It might in the future though, I guess